submissions/YarHadi/oop_exercise/index.js

```diff
@@ -0,0 +1,55 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/YarHadi/a-tiny-JS-world
+   Web app: https://yarhadi.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+   name;
+   gender;
+   saying;
+ 
+   constructor(name, gender, saying) {
+     this.name = name;
+     this.gender = gender;
+     this.saying = saying;
+   }
+   show() {
+     const keys = ["species", "name", "gender", "legs", "hands", "saying"];
```

Comment:
A class should be agnostic of other classes' properties. Encapsulation (OOP) and Open/Closed principle (SOLID).